src/blister.h

```diff
@@ -209,29 +209,88 @@ static inline UInt * BLOCKS_BLIST(Obj list)
 /****************************************************************************
 **
 *F  COUNT_TRUES_BLOCK( <block> )  . . . . . . . . . . . count number of trues
+*F  COUNT_TRUES_BLOCKS( <ptr>, <nblocks> ) 
+**
+**  This function uses either the gcc builtin __builtin_popcount which
+**  generates the popcntl or popcntq instruction on sufficiently recent intel
+**  CPUs or some alternative code otherwise, or the algorithm below.
```

Comment:
This is a bit difficult to read, as the two "or" close different phrases. Perhaps change the last "or" to "or if __builtin_popcount is not available, the algorithm below."

Perhaps a bigger change would be better, though, e.g.: "This function uses either the gcc builtin __builtin_popcount, or the algorithm described below. If __builtin_popcount is used, then on sufficiently recent Intel CPUS, the compiler may generate code using the popcntl or popcntq instructions, or equivalent SSE/AVX code."